rebar.config

```diff
@@ -1,6 +1,91 @@
-{erl_opts, [debug_info,
-            {platform_define, "R1[45]", no_binary_to_integer}]}.
-{eunit_opts, [verbose, {report,{eunit_surefire,[{dir,"."}]}}]}.
+%% -*- mode: erlang;erlang-indent-level: 2;indent-tabs-mode: nil -*-
+%% ex: ts=4 sw=4 ft=erlang et
+
+%% == Erlang Compiler ==
+
+%% Erlang compiler options
+{erl_opts, [ warn_unused_vars
+           , warn_export_all
+           , warn_shadow_vars
+           , warn_unused_import
+           , warn_unused_function
+           , warn_bif_clash
+           , warn_unused_record
+           , warn_deprecated_function
+           , warn_obsolete_guard
+           , strict_validation
+           , warn_export_vars
+           , warn_exported_vars
+           , warn_missing_spec
+           , warn_untyped_record
+           , debug_info
+           , {platform_define, "R1[45]", no_binary_to_integer}]}.
+
+{profiles, [
+  {test, [
+    {deps, [ {katana_test, {git, "https://github.com/inaka/katana-test.git", {tag, "0.0.5"}}}
+           , {mixer,       {git, "https://github.com/inaka/mixer.git",       {tag, "0.1.5"}}}
+           , {meck,        "0.8.4"}
+           , {xref_runner, {git, "https://github.com/inaka/xref_runner",     {tag, "0.2.6"}}}
+    ]}
+  ]},
+  {shell, [
+    {deps, [
+      {sync, {git, "https://github.com/rustyio/sync.git", {ref, "9c78e7b"}}}
```

Comment:
remove this dependency if possible, (maybe leave it in as a recommended and commented entry?)